terraform/globals.tf

```diff
@@ -57,13 +57,13 @@ variable "vagrant_cidr" {
 }
 
 variable "dns_zone_id" {
-  description = "Amazon Route53 DNS zone identifier"
-  default     = "Z3SI0PSH6KKVH4"
+  description = "Amazon Route53 DNS zone identifier. Different per account."
+  default     = "dummy_default"
 }
 
 variable "dns_zone_name" {
-  description = "Amazon Route53 DNS zone name"
-  default     = "cf.paas.alphagov.co.uk"
+  description = "Amazon Route53 DNS zone name. Differs per account."
+  default     = "dummy_default"
```

Comment:
Do we want these default values here? I'm concerned that there's a risk these could leak into a real environment.

The commit message mentions running terraform with a limited scope, and therefore not having to specify these. I believe terraform will still update the outputs when running with a limited scope, and therefore these could end up in the outputs, and get written into the S3 bucket.